src/plugins/data-card/data-card-content.ts

```diff
@@ -180,9 +190,12 @@ export const DataCardContentModel = TileContentModel
         }
         else {
           if (!sharedDataSet) {
-            // The document doesn't have a shared model yet
-            const dataSet = defaultDataSet();
+            // The document doesn't have a shared model yet; create one.
+            const tile = getTileModel(self);
+            const dataSet = defaultDataSet(tile!.title);
```

Comment:
Same question as with the table code about using the tile model title temporarily.